Linux: Click to restart and update Zed does nothing

### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

As the topic states. Clicking on the status bar does not result in any action; ZED update is expected.

### Environment

Zed: v0.143.6 (Zed)
OS: Linux X11 ubuntu 24.04
Memory: 15.3 GiB
Architecture: x86_64

### If applicable, add mockups / screenshots to help explain present your vision of the feature

Functionality should mirror MacOS behavior and update Zed.

### If applicable, attach your Zed.log file to this issue.

Two clicks result in following log.

<details><summary>Zed.log</summary><pre>
```
2024-07-12T03:45:20-05:00 [ERROR] oneshot canceled
2024-07-12T03:45:21-05:00 [ERROR] oneshot canceled
```

--- a/crates/gpui/src/platform/linux/platform.rs
+++ b/crates/gpui/src/platform/linux/platform.rs
@@ -184,7 +184,7 @@ impl<P: LinuxClient + 'static> Platform for P {
         // cleaned up when `kill -0` returns.
         let script = format!(
             r#"
-            while kill -O {pid} 2>/dev/null; do
+            while kill -0 {pid} 2>/dev/null; do
                 sleep 0.1
             done
 

